### ISSUE TITLE
About the VFL loss

### ISSUE THREAD

**Issue by whuxfx** [xiuqhou/Relation-DETR#26]
### Question

In Relation DETR, is the VFL loss the same as the vari_sigmoid_focal_loss function?

**Comment by xiuqhou** [comment]
yes, vari_sigmoid_focal_loss is used in hybrid set criterion for Relation DETR
